views/layouts/main.njk

```diff
@@ -16,6 +16,10 @@
  
     {% endblock %}
 
+    {% block scripts %}
```

Comment:
Is this needed?

Reply:
Yes, this essentially is chunk of nunjucks in the header that can be overwritten by other files. Without it there wouldn't be a way to put scripts in the header for a specific file.